### DISCUSSION TITLE
Include API testing in workflows

### DISCUSSION THREAD

**Discussion by freddydk** [microsoft/AL-Go#110] [Ideas]
Add functionality to support
- Adding postman API test collections (like test apps)
- Automatically use postman tools to execute and verify the API's exposed by your app
- https://www.postman.com/automated-testing/
- Publish API test results

**Comment by sstorie** [comment]
This is quite old, but I am looking at AL-Go now and curious how best to integrate Playwright tests into my actions. We primarily develop OData integrations so confirming the functionality of our entities is something we're trying to add.
